.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,13 @@
-For plugin updates, please indicate which repos this should be built into:
+If your package needs to be released within one or more release streams, and/or distributions, please open PRs to each of those branches respectively. The easiest way to do this is to make the initial commit for the mainline branch (e.g. rpm/develop or deb/develop) and then cherry pick the commit hash onto each subsequent branch.
 
-* [x] Nightly
-* [ ] 1.20
-* [ ] 1.19
-* [ ] 1.18
+Example:
+
+    git checkout -b foreman-tasks-1.0.1 rpm/develop
```

Comment:
I think for DEBs, the branch name still has to start with e.g. `deb/develop` and `deb/1.20` for the automated PR tests to work, so it would be nice to have it like this in the example, then `rpm` has just to be replaced with `deb` and all will be fine.